scripts/sysman/power.yml

```diff
@@ -280,6 +288,7 @@ details:
     - "The application may call this function from simultaneous threads."
     - "The implementation of this function should be lock-free."
     - "1.5": "Note: This function is deprecated and replaced by $sPowerSetLimitsExt."
+    - "1.7": "Note: $sPowerSetLimitsExt function is deprecated and functionality is provided by $sPowerSetPowerLimits."
```

Comment:
same for 1.5 and 1.7 as above

Reply:
same as above, need to define new func before deprecating. fixed the 1.5 line